simtools/model/camera.py

```diff
@@ -25,12 +25,12 @@ class Camera:
         assumed to be in a sim_telarray format.
     getDiameter()
         Get pixel diameter.
-    getFunnelShape()
-        Get funnel shape.
-    getFunnelEfficiencyFile()
-        Get funnel efficiency filename
-    getFunnelWavelengthFile()
-        Get funnel wavelength file contained in _pixels
+    getPixelShape()
+        Get pixel shape.
+    getLightguideEfficiencyAngle()
```

Comment:
I think it's better to keep "file" in the name of the function, like getLightguideEfficiencyAngleFilename.

Reply:
OK, will add Filename, no problem.